docs/ops_guide/system_stores.md

```diff
@@ -0,0 +1,69 @@
+should not matter, but it can be useful for operators to know about them.
+
+This page explains the various types of system stores and their function.
+
+## System Store Types
+
+Below are a few distinct types of system stores:
+
+| Type                   | Cardinality    | Content       |
+|------------------------|----------------|---------------|
+| Schema system store    | 1 / deployment | Metadata only |
+| Global system store    | 1 / deployment | Data          |
+| Cluster system store   | 1 / cluster    | Data          |
+| Per-store system store | 1 / user store | Data          |
+
+### Schema System Stores
+
+These are stores created in just one of the clusters of a deployment. They contain no data at all (and therefore no 
+store-versions) and thus cannot be queried. They only serve to store schemas. These schemas are those which the system
+needs for its own [internal protocols](https://venicedb.org/javadoc/com/linkedin/venice/serialization/avro/AvroProtocolDefinition.html).
+
+The schema system stores are used for the sake of protocol forward compatibility. When deploying Venice code, its jars 
+contain resource files for the current protocol versions as well as all previous protocol versions, but not future 
+protocol versions. Given that Venice is a distributed system, it is important to have the flexibility of deploying 
+different versions of the software across different nodes. When a node running a newer version of the software persists
+data written with a newer protocol version, and that data is then read by another node running an older version of the
+software which does not know about that newer protocol version, then it can query the schema of the appropriate system
+store and use this to decode the newer payload, and translate them into the version they understand (a process known as
+Schema Evolution).
+
+### Global System Stores
+
+These are stores created in just one of the clusters of a deployment, but in opposition to schema system stores, they
+do contain data. Their global nature makes them convenient to store data about all clusters, but it also means that the
+clusters are not isolated from one another as far as this kind of system store is concerned. Therefore, it is best for
+this type of system store to serve only in "best effort" purposes, so that if for any reason they become unavailable,
+critical operations can still continue. An example of this would be to store heuristics or other non-critical data.
+
+### Cluster System Stores
+
+These are stores created once per cluster, and which do contain data. It is appropriate to use such stores for critical
+operations, given that one cluster's availability issues would not bleed into other clusters.
+
+### Per-store System Stores
+
+These are stores created once per user store, and which do contain data.
+
+## System Store List
+
+Below is a list of actual system stores, along with their type and function (excluding schema stores):
```

Comment:
I was wondering whether its better, if we split the examples of various system stores into the subsections in the earlier paragraphs instead of all them in a single para.

Reply:
I'm not sure... for someone wanting to learn more about what some specific system store does, but without knowing what type it belongs to, then I feel like having the full list is more useful. But I added a small sub-list of system stores belonging to each type in the type description section. See the latest commit. Maybe that's fine?

Reply:
Yes, that looks fine. Thanks